task1/src/redux/reducer/albumsReducer.js

```diff
@@ -0,0 +1,63 @@
+import { ADD_ALBUM_FAILURE, ADD_ALBUM_STARTED, ADD_ALBUM_SUCCESS, GET_ALBUMS_FAILURE, GET_ALBUMS_STARTED, GET_ALBUMS_SUCCESS, SET_ACTIVE_ALBUM, SET_ALBUMS } from "../const/const";
+
+const initialState = {
+    albums: [],
+    albumsLoaded: true,
+    albumAdded: true,
+    additionalAlbums: [],
+    albumsLoadError: null,
+    addAlbumError: null,
+    activeAlbumInfo: { albumId: null, photos: [] },
+}
+
+export default function albumReducer(state = initialState, action) {
+    switch (action.type) {
+        // get albums reducers
+        case GET_ALBUMS_STARTED:
+            return {
+                ...state,
+                albumsLoaded: false,
+            }
+        case GET_ALBUMS_SUCCESS:
+            return {
+                ...state,
+                albumsLoaded: true,
+                albums: action.payload,
+            }
+        case GET_ALBUMS_FAILURE:
+            return {
+                ...state,
+                albumsLoaded: true,
```

Comment:
are albums loaded when we call action `GET_ALBUMS_FAILURE` ?

Reply:
Of course the main idea i've got an error, i can rename this field to "albumsLoading" but because of this i will need to change true to false & false to true awerywhere, may i leave albumsLoaded? "albumsLoaded" can also mean than i got zero albums because of as error

Reply:
you may, but it is not good